src/wallet/scriptpubkeyman.cpp

```diff
@@ -265,6 +265,31 @@ bool LegacyScriptPubKeyMan::EncryptKeys(CKeyingMaterial& vMasterKeyIn)
     return true;
 }
 
+bool LegacyScriptPubKeyMan::GetReservedDestination(const OutputType type, bool internal, int64_t& index, CKeyPool& keypool)
+{
+    {
```

Comment:
why the scope here?

Reply:
A lock is going to be added in the next step

Reply:
> A lock is going to be added in the next step

The scope isn't needed even with the lock and all other changes from #17261:

https://github.com/bitcoin/bitcoin/blob/35b0f7b5447bce4eef360c7e4237a928bb0eba3c/src/wallet/scriptpubkeyman.cpp#L275-L290

But I don't think it's a big deal. Could clean this up later if reservekey and reservedestination methods are merged later (https://github.com/bitcoin/bitcoin/pull/17304#discussion_r340311958)

Reply:
I'll just leave this as is so it can get merged sooner :)